src/walletSdk/Anchor/index.ts

```diff
@@ -128,15 +133,18 @@ export class Anchor {
 
     try {
       // TODO - use httpClient
```

Comment:
Should we use client here?

Reply:
yup good catch https://github.com/stellar/typescript-wallet-sdk/pull/8/commits/d657eea6d56447f306ef5d2dd90e0bd3b815c7cc